sync/class.jetpack-sync-module-posts.php

```diff
@@ -294,7 +294,7 @@ public function is_gutenberg_meta_box_update() {
 			'editpost' === $_POST['action'] &&
 			'1' === $_GET['classic-editor'] &&
 			'1' === $_GET['meta_box'] &&
-			Jetpack::is_gutenberg_available()
+			Jetpack_Gutenberg::is_gutenberg_available()
```

Comment:
Should we rename this to `Jetpack_Gutenberg::is_available()` ?

Reply:
`is_gutenberg_available` leave no room for confusion. Let's just leave it.